packages/typespec-vscode/src/vscode-cmd/emit-code/emit-code.ts

```diff
@@ -460,6 +461,13 @@ async function doEmit(
           });
           return ResultCode.Fail;
         }
+        const isSupport = await isCompilerSupport(tspLanguageClient);
+        if (!isSupport) {
+          logger.info(
+            "Compile project is not supported by the current TypeSpec Compiler's LSP. Emitting Failed.",
```

Comment:
A bit confused by this log.  There is no 'Compile Project' command in TypeSpec extension and I don't think end user understand about LSP,  could you make the log info more clear to the end user?